Makefile

```diff
@@ -28,6 +31,6 @@ piglet-matcher-test: piglet-matcher.c piglet-matcher-test.c
 	./piglet-matcher-test
 
 clean:
-	rm -vf $(INTERPRETERS) pigletvm pigletvm-test piglet-matcher piglet-matcher-test
+	rm -vf $(INTERPRETERS) regexp-interpreter pigletvm pigletvm-test piglet-matcher piglet-matcher-test
```

Comment:
All part 1 interpreters are listed in the $(INTERPREteRS)

I suggest just adding interpreter-regexp to the $(INTERPRETERS) list, to get uniform handling at least in the first set of examples:

I.e., just do `INTERPRETERS = basic-switch immediate-arg stack-machine register-machine regexp`.